tarantool_test.go

```diff
@@ -708,7 +708,9 @@ func TestTtDialer(t *testing.T) {
 	assert := assert.New(t)
 	require := require.New(t)
 
-	conn, err := TtDialer{}.Dial(server, DialOpts{})
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
```

Comment:
What about to create a test_helpers function to get a `Connect` context timeout for unite/integration-tests (but not for examples because it would be confusing for an example reader)?
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), test_helpers.GetTestConnectTimeout())
	defer cancel()
```

It would help us to change the timeout for all tests in the future.

Reply:
Will it be ok to add 2 helpers: `GetTestConnectTimeout` and `GetTestPoolConnectTimeout` (to manage pool connections)? In case we want to increase timeout for pool creations only.

Reply:
Updated (constant for `connection.Connect` is in `test_helpers/main.go`, for `pool.Connect` is in `test_helpers/pool_helper.go`).